content/en/tracing/service_catalog/guides/integrating-with-ci-visibility.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Integrating with CI Visibility
+kind: guide
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+In the Software Delivery lens you can see data from your pipeline executions associated to your services. 
+
+By default, the pipelines are associated to your service by matching the repository URLs with the ones in [CI Visibility][4]. 
+However, you might want more control into what pipelines match with your services. You can configure the pipelines associated 
```

Comment:
You can edit the pipelines associated with each service by modifying the `ci-pipeline-fingerprints` field in your [service metadata][5].